### DISCUSSION TITLE
Calibration Mode: Is 15% minimum charge sufficient to calibrate the battery?

### DISCUSSION THREAD

**Discussion by qfettes** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#456] [General]
The new feature to automatically perform a battery cycle is useful, but I'm wondering if discharging to only 15% is sufficient to calibrate the battery? Previously, I was using this guide: https://help.ifixit.com/article/265-battery-calibration, which recommends discharging the battery to 0% and allowing it to stay in that state for a few hours. Thanks in advance for looking into this.

**Comment by MatthiasKerbl** [comment]
> The new feature to automatically perform a battery cycle is useful, but I'm wondering if discharging to only 15% is sufficient to calibrate the batter? Previously, I was using this guide: https://help.ifixit.com/article/265-battery-calibration, which recommends discharging the battery to 0% and allowing it to stay in that state for a few hours. Thanks in advance for looking into this.

Hi @qfettes,

As always, there is not only black and white. Calibrating your battery with our Calibration Mode is definitely better than not doing it. However, it is probably even better if you do it as described in the article you mentioned.
We have to find somewhat of a middle ground between convenience and effectiveness but we are definitely open to improving the Calibration Mode further.

Cheers,
Matthias
